packages/rspack/tests/Stats.test.js

```diff
@@ -176,7 +176,6 @@ describe("Stats", () => {
 		<t> finish make hook: X ms
 		<t> finish compilation: X ms
 		<t> seal compilation: X ms
-		<t> afterCompile hook: X ms
```

Comment:
Forgot this line. Is this intentional?

Reply:
it's intentional, we should log this in js side after remove it from rust side, but afterCompile is not affected for performance, so it should be ok to just remove it